### DISCUSSION TITLE
Videos have wrong preview thumbnail

### DISCUSSION THREAD

**Discussion by notyourattorney** [photoprism/photoprism#2423] [Q&A] [answered]
Across several instances, I have noticed videos have a preview thumbnail that is simply wrong and unrelated to the video. I have experienced this error on Docker/PhotoPrism on a QNAP NAS (container station), as well as on an iMac (docker desktop) using mapped network drives over Thunderbolt3.

In one case, the preview image is one I know is just a photo that inexplicably became the thumbnail for a video. When I click on it the picture disappears and the video plays. When I move the slider to restart the video the first frame is the video as expected. Hence it is only the preview thumbnail that is wrong.

I am currently rescanning the library to see if that rebuilds the video preview correctly. If there is a command to update the preview image for videos please let me know as I've only come across the thumbs command.

**Comment by notyourattorney** [comment, marked as answer]
LASTZERO's COMMENT IS THE ANSWER

------------------------------------------
I uploaded a partial of the wrong thumbnail image and the actual video first frame. 

<img width="318" alt="Screen Shot 2022-06-14 at 6 15 56 PM" src="https://user-images.githubusercontent.com/50841636/173715583-0a98514d-1657-4c6d-b9cc-33f6b6f2cb80.png">

<img width="1215" alt="Screen Shot 2022-06-14 at 6 13 22 PM" src="https://user-images.githubusercontent.com/50841636/173715384-5722d74b-d27d-467d-9e80-b065cc4e5e9f.png">

**Reply by lastzero** [reply]
Could it be there is a JPEG file in the same folder and with the same name as the video? In that case, it would be used as preview for the video as explained in the docs.

**Reply by notyourattorney** [reply]
That is exactly what happened!! After some looking, I determined it was a Google photos directory with an enormous number of files so the file numbering repeated. Here there was a JPEG image with the same file name as a video. The odd thing (from a matching them perspective) is these two items are from different years and different cameras there's nothing about them that's the same in terms of the hardware that generated them. 

I will try to move the various file types into separate directories to prevent that, and reindex everything. If there is an easier way via an option, please let me know.

Thanks again!
